includes/pipeline_page/docs_schema.php

```diff
@@ -58,6 +58,13 @@ function print_param($is_group, $param_id, $param, $is_required=false){
           '.parse_md($param['help_text'])['content'].'</div>';
     }
 
+    # default value 
+    $default_val = '';
+    if(array_key_exists("default", $param) && strlen(trim($param['default'])) > 0){
+      $default_val = is_string($param['default'])? "'".$param['default']."'":$param['default'];
+      $default_val = '<code class=""><span class="text-muted">default:</span>'.$default_val.'</code>';
```

Comment:
Do we need the span? Can the text-muted not go in the `<code>` tag?

Reply:
If I add `.text-muted` to the whole code tag, the default value itself will also be muted. I think it helps to parse the docs faster, if the default value is highlighted a bit.

Reply:
Ah of course 👍🏻 

Ok, then why the empty `class=""` on the `<code>`?